code/processes/filealerter.q

```diff
@@ -41,7 +42,23 @@ find:{[path;match]
 	.lg.o[`alerter;"searching for ",path,"/",match];
 	files:@[system;findstring;()];
 	if[os=`win;files:,/:[path,"\\"; files]]; files};
+
+//-decodes pcap file and sends to tickerplant
+processpcaps:{[path;file;pcaptab]
+	.lg.o[`alerter;"processing pcap file"];
+	.lg.o[`alerter;"decoding pcap file"];
+	table: .pcap.buildtable[hsym `$(path,"/",file)];
 	
+	.lg.o[`alerter;"checking connection to tickerplant"];
+	sendtotickerplant[tickerplanttype;pcaptab;table[cols table]]
+	}
+
+sendtotickerplant:{[tptype;t;x]
+	if[.servers.gethandlebytype[tptype;`any]~`int$();
```

Comment:
Could you make this 

if[count .servers.gethandlebytype[tptype;`any]

it's more supportable this way as if we were to change the handle list to be a different type for some reason then this would succeed either way